core/storage/audit/gae_models.py

```diff
@@ -22,6 +22,12 @@
 from core.platform import models
 import feconf
 
+from typing import Any, Dict, Text # isort:skip # pylint: disable=unused-import
```

Comment:
Any is not used anywhere, btw I also realized, can this import be moved to the if block below.

Reply:
Actually we might be using casts in our codebase. And in future we will be migrating to py3 type annotations style. So this will require the classes to be imported while runtime also.